Lesson_27/Homework_27.py

```diff
@@ -0,0 +1,63 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+import time
+
+
+class NovaPoshtaTracking:
```

Comment:
It is better to use PageObject patter for ui test automation. Driver initialization should be better inside fixture.